gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/push/PushSubscriberHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.gcp.pubsub.push;
+
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.pubsub.v1.ProjectSubscriptionName;
+import io.micronaut.http.MutableHttpResponse;
+import reactor.core.publisher.Mono;
+
+/**
+ * Handles incoming {@link PushRequest} messages.
+ *
+ * @author Jeremy Grelle
+ * @since 5.4.0
+ */
+public interface PushSubscriberHandler {
```

Comment:
Add nullabiliy annotations to method and return types.

Reply:
[ae41c3c](https://github.com/micronaut-projects/micronaut-gcp/pull/1012/commits/ae41c3cb72f3a4d2ef48e2fdd6979793f49f0a32)